docs/Running-behind-nginx.md

```diff
@@ -1,9 +1,43 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx
 
-To pass Netdata via a nginx, use this:
+## Intro
+
+[Nginx](https://nginx.org/en/) is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server used to host websites and applications of all sizes. 
+
+The software is known for its low impact on memory resources, high scalability, and its modular, event-driven architecture which can offer secure, predictable performance.
+
+## Why Nginx
+
+Nginx is used and useful in cases when you do not want to access Netdata via the default `IP ADDRESS:19999`.
+
+With Nginx, the data is accessed via human-readable domains such as `example.com` or `sub.example.com`. 
+
+By default, Nginx is fast and lightweight out of the box.
+
+Nginx can be configured to secure your Netdata insights by the use of authentication to access the dashboard. 
+
+### Nginx configuration file
```

Comment:
This doesn't belong under Why Nginx.

Reply:
Yes, adding that to /etc/hosts also works. 

I have removed the sentence, and paraphrased it under ["Enable authentication"](https://docs.netdata.cloud/docs/running-behind-nginx/#enable-authentication)

Reply:
Actually, I was referring to the heading level of 'Nginx configuration file'. We have it as a child of 'Why Nginx', which isn't correct. You probably want to make it the same level as 'Why Nginx'.